.github/workflows/publish.yml

```diff
@@ -15,12 +15,47 @@ jobs:
       go-version: '1.23'
       go-lint-version: 'v1.60.2'
       run-unit-tests: true
-      run-integration-tests: true
       run-lint: true
       run-build: true
       run-gosec: true
       gosec-args: "-exclude-generated -exclude-dir=itest -exclude-dir=testutil ./..."
 
+  e2e_babylon:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.23'
+      - name: Run e2e Babylon tests
+        run: make test-e2e-babylon
+
+  e2e_bcd:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.23'
+      - name: Run e2e BCD tests
+        run: make test-e2e-bcd
+
+# TODO: disable e2e_op as removing fast sync might introduce nontrivial change
```

Comment:
Do we have an issue to track it? Seems critical

Reply:
This is fixed actually. Reenabled